$NetBSD: patch-src_runtime_config.arm-bsd,v 1.1 2019/11/10 16:58:31 he Exp $

Use Config.generic-bsd.
Don't set, but add to common vars.

--- src/runtime/Config.arm-bsd.orig	2019-09-28 09:49:01.000000000 +0000
+++ src/runtime/Config.arm-bsd
@@ -9,19 +9,18 @@
 # provided with absolutely no warranty. See the COPYING and CREDITS
 # files for more information.
 
+include Config.generic-bsd
+
 ASSEM_SRC = arm-assem.S
 ARCH_SRC = arm-arch.c
 
-OS_SRC = bsd-os.c arm-bsd-os.c
+OS_SRC += arm-bsd-os.c
 
 ifdef LISP_FEATURE_GENCGC
   GC_SRC = fullcgc.c gencgc.c traceroot.c
 else
   GC_SRC = cheneygc.c
 endif
-ifdef LISP_FEATURE_SB_CORE_COMPRESSION
-  OS_LIBS += -lz
-endif
 
 # Nothing to do for after-grovel-headers.
 .PHONY: after-grovel-headers
